src/components/ui/interaction/Clickable.tsx

```diff
@@ -60,8 +64,9 @@ export class ClickableComponent extends React.Component<ClickableInnerProps> {
     const hasClickHandler = !!(onClick || onDblClick);
     const opacity = !disableOpacityOnClick && mouseIsDown ? 0.5 : undefined;
     return (
-      <div
-        className={opacityOnHover ? hoverStyle : undefined}
+      <
+
```

Comment:
What happened here? No `div`?

Reply:
All usage of className and `css` has been replaced with `styled`.

Reply:
Not a huge fan of the resulting code.

```
const HoverStyledDiv = setDisplayName<
  HoverStyledDivProps &
    React.DetailedHTMLProps<
      React.HTMLAttributes<HTMLDivElement>,
      HTMLDivElement
    >
>('HoverStyledDiv')(styled.div<HoverStyledDivProps>`
  :hover {
    ${props => (props.opacityOnHover ? 'opacity: 0.7;' : '')};
  }
`);
```

Reply:
Fixed :)